force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html

```diff
@@ -156,17 +156,14 @@
                                 {item.recurringDonation.npe03__Last_Payment_Date__c}</div>
                         </td>
                         <td key={item.nextDonation} data-id={item.recurringDonation.Id} class={tdClasses}  data-label={labels.nextDonation}>
-                            <div class={rowClasses} style={fixedWidth}> 
-                                {item.nexDonationFormatFirstElement} </br>
-                                {item.nexDonationFormatSecondElement}
-                            </div>
+                            <div class="next-donation" lwc:dom="manual"></div>
                         </td>
                         <td key={item.lastModified} data-id={item.recurringDonation.Id} class={tdClasses}  data-label={labels.lastModified}>
-                            <div class={rowClasses} style={fixedWidth} title="Last Modified">
+                            <div class={rowClasses} style={fixedWidth} title={labels.lastModified}>
                                 {item.lastModified}</div>
                         </td>
-                        <td data-label="Actions">
-                            <lightning-button-menu class="dv-dynamic-mobile" menu-alignment="auto" alternative-text="Action menu">
+                        <td data-label={labels.RD2_Actions}>
```

Comment:
`data-label` is not UI text. It's strictly an internal html dom attribute and does not require (and probably should not be) dynamic text.

Reply:
@lparrott is it possible you continue with Michael’s change request and review Mauricio's reply ? Michael is on PTO during the next week and we would appreciate to finish with this PR during the first days of the next week.
FYI @gerardorodriguezsf

Reply:
If this Label ("Actions") actually shows up in the UI, then what you did is fine.